package.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "vue-adjmatrix",
+  "name": "MultiMatrix",
   "version": "0.1.0",
   "private": true,
-  "description": "An adjacency matrix visualization built to fit in with multinet",
+  "description": "A first class MultiNet adjacency matrix visualization",
```

What do you mean by `first class` here? I would suggest:

```suggestion
  "description": "An adjacency matrix visualization for MultiNet",
```